### DISCUSSION TITLE
Runelite client blurry, but gameplay is fine?

### DISCUSSION THREAD

**Discussion by amirtcu** [runelite/runelite#17005] [Q&A]
I started experiencing this overnight, but it seems the Runelite Client is blurry in everywhere but the gameplay. For example, the image when the client starts up is blurry, the text in the sidebar is blurry, etc. The gameplay itself is OK, but I can't figure out why the text in the sidebar got blurry? Computer settings are fine - the issue seems to be purely isolated to Runelite. Even when the client is starting up, the image is blurry. 

I've tried uninstalling and reinstalling the client but that hasn't done anything. Screenshot attached.

<img width="206" alt="Screenshot 2023-09-09 105448" src="https://github.com/runelite/runelite/assets/144549020/dde9b9e7-a8a9-4501-a482-08d243337d0d">

**Comment by AdpTShadw** [comment]
Find the solution? Im having the same issue atm
